src/python/blazingmq/dev/it/process/broker.py

```diff
@@ -34,7 +34,7 @@ class Broker(blazingmq.dev.it.process.bmqproc.BMQProcess):
     def __init__(self, config: cfg.Broker, cluster, **kwargs):
         cwd: Path = kwargs["cwd"]
         (cwd / "bmqbrkr.ctl").unlink(missing_ok=True)
-        cwd.joinpath("storage", "archive").mkdir(parents=True)
+        cwd.joinpath("storage", "archive").mkdir(parents=True, exist_ok=True)
```

Comment:
@jll63 note that this is needed to restart the cluster with the same configs/work folder/storage.